.env.example

```diff
@@ -52,3 +52,7 @@ HYPOTHESIS_PROFILE=dev
 
 VOYAGER_API_URL=
 VOYAGER_API_KEY=
+
+# Private key to sign transactions for RLP scripts/compute_rlp_encoding.ts
+# Warning: Do not use this key in production systems
+PRIVATE_KEY_RLP_SCRIPT=0x6be0067ba259624aa28f796b703e7d095925a470b17786767bd09aaa3fc2ceea
```

Comment:
why not just hard coding this in the script with a random value? also I think that the script should be removed in favor a a python one using tools we use for testing (and better: migrated to exec spec)

Reply:
> why not just hard coding this in the script with a random value? also I think that the script should be removed in favor a a python one using tools we use for testing (and better: migrated to exec spec)

eventually, yes, next week I will start looking into proper test data generation